x-pack/packages/kbn-elastic-assistant/impl/assistant_context/constants.tsx

```diff
@@ -13,6 +13,11 @@ export const SYSTEM_PROMPT_LOCAL_STORAGE_KEY = 'systemPrompts';
 export const LAST_CONVERSATION_ID_LOCAL_STORAGE_KEY = 'lastConversationId';
 export const KNOWLEDGE_BASE_LOCAL_STORAGE_KEY = 'knowledgeBase';
 
+/** The default `n` latest alerts, ordered by severity, sent as context to the assistant */
```

Comment:
```suggestion
/** The default `n` latest alerts, ordered by risk score, sent as context to the assistant */
```

Isn't it by risk score?

Reply:
Yes, the alert counts tool is grouped by `kibana.alert.severity` (see `x-pack/plugins/elastic_assistant/server/lib/langchain/tools/alert_counts/get_alert_counts_query.ts`), but the open alerts tool is ordered by `kibana.alert.risk_score` (see `x-pack/plugins/elastic_assistant/server/lib/langchain/tools/open_alerts/get_open_alerts_query.ts`)